conjure_oxide/tests/integration/minion_constraints/diseq/input.eprime

```diff
@@ -0,0 +1,7 @@
+language ESSENCE' 1.0
+
+find x: int(1..10)
```

Comment:
maybe give them smaller domains unless the larger domain demonstrates something important?